manifests/maintenance/pg_repack.pp

```diff
@@ -30,6 +30,7 @@
   $other_tables    = '-t producers -t resource_params -t resource_params_cache"'
   $reports_table   = '-t reports"'
   $resource_events_table = '-t resource_events"'
+  $catalog_inputs_table = '-t catalog_inputs"'
```

Comment:
I would probably add this to the existing pg_repack catalogs tables cron job unless we think there's a specific reason it needs to be separated.  

you could implement it something like this was when I added resource_events ( when we were still doing reports and resource_events ) ... 

https://github.com/puppetlabs/puppetlabs-pe_databases/pull/28/files#diff-16594a284365491882a33bbff5491440R30

However, if there's a reason to keep it separate I'm open to that discussion as well.  If it joins to the existing catalogs tables it seems like a good reason to make sure they are all repacked together.

Reply:
Yeah that sounds reasonable to me, I'll update the PR